crates/oci-distribution/src/lib.rs

```diff
@@ -252,4 +289,24 @@ mod test {
         assert_eq!(manifest.schema_version, 2);
         assert!(!manifest.layers.is_empty());
     }
+
+    #[tokio::test]
+    async fn test_pull_layer() {
+        let image = Reference::try_from(HELLO_IMAGE).expect("parsed reference");
+        let mut c = Client::default();
+        c.auth(&image, None).await.expect("authenticated");
+        let manifest = c.pull_manifest(&image).await.expect("pull manifest");
+
+        // Pull one specific layer
+        let file: Vec<u8> = Vec::new();
+        let layer0 = manifest.layers[0].clone();
```

Comment:
The `clone` here isn't neccessary. You can just borrow like this: `&manifest.layers[0]`

Reply:
Fixed